presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergResourceFactory.java

```diff
@@ -58,9 +59,10 @@ public class IcebergResourceFactory
     private final List<String> hadoopConfigResources;
     private final NessieConfig nessieConfig;
     private final S3ConfigurationUpdater s3ConfigurationUpdater;
+    private final GcsConfigurationInitializer gcsConfigurationInitialize;
 
     @Inject
-    public IcebergResourceFactory(IcebergConfig config, IcebergCatalogName catalogName, NessieConfig nessieConfig, S3ConfigurationUpdater s3ConfigurationUpdater)
+    public IcebergResourceFactory(IcebergConfig config, IcebergCatalogName catalogName, NessieConfig nessieConfig, S3ConfigurationUpdater s3ConfigurationUpdater, GcsConfigurationInitializer gcsConfigurationInitialize)
```

Comment:
Since we are injecting `GcsConfigurationInitializer`, we need to bind `com.facebook.presto.hive.gcs.HiveGcsModule` in `com.facebook.presto.iceberg.InternalIcebergConnectorFactory`